PyPoE/cli/exporter/wiki/parsers/item.py

```diff
@@ -2361,17 +2393,17 @@ def _currency_extra(self, infobox, base_item_type, currency):
     )
 
     _master_hideout_doodad_map = (
-        ('HideoutNPCsKey', {
-            'template': 'master',
-            'format': lambda v: v['Hideout_NPCsKey']['Name'],
-            'condition': lambda v: v is not None,
-        }),
-        ('MasterLevel', {
-            'template': 'master_level_requirement',
-        }),
-        ('FavourCost', {
-            'template': 'master_favour_cost',
-        }),
+        # ('HideoutNPCsKey', {
+        #     'template': 'master',
+        #     'format': lambda v: v['Hideout_NPCsKey']['Name'],
+        #     'condition': lambda v: v is not None,
+        # }),
+        # ('MasterLevel', {
+        #     'template': 'master_level_requirement',
+        # }),
+        # ('FavourCost', {
+        #     'template': 'master_favour_cost',
+        # }),
```

Comment:
Is this a case of YAGNI?

Will we ever have any need for this? @Journeytojah @angelic-knight 
If not needed, just remove outright instead of comment. But please check with one another first.

Reply:
I think this should be safe to delete. @Journeytojah, what's your take on it?

Reply:
This is fine to remove.

Reply:
I've removed this and surrounding related code.